pmultiqc/modules/common/histogram.py

```
@@ -118,6 +120,97 @@ def add_value(self, value, stack="total"):
+                    val_str = str(int(val) if isinstance(val, (int, np.integer, float)) and float(val).is_integer() else val)
+                    if val_str in self.data:
+                        self.data[val_str][stack] += 1
+
+                # Add above threshold count
+                if above_count > 0:
+                    self.out_threshold = True
+                    self.data[self.bins[-1]][stack] += above_count
+            else:
+                # No breaks - use value_counts approach
+                unique, counts = np.unique(arr, return_counts=True)
+                for val, count in zip(unique, counts):
```

Ruff (0.14.13)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)